### PR TITLE
Added Parse-server example.

### PR DESCRIPTION
**Pull request by jmdobry**
@JustinBeckwith

### CODE REVIEW

**Comment by JustinBeckwith** [comment]
Nice!  LGTM

**Comment by JustinBeckwith** [comment]
Do you have this running on the docs-and-samples project yet?  Link?

**Comment by jmdobry** [comment]
https://parse-dot-nodejs-docs-samples.appspot.com/

**Comment by jmdobry** [comment]
Ready for review

File: appengine/parse-server/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+This sample demonstrates deploying a [Parse-server](https://github.com/ParsePlatform/parse-server)
+app to [Google App Engine Managed VMs](https://cloud.google.com/appengine).
+
+## Setup
+
+1. Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com/).
+1. [Enable billing](https://console.cloud.google.com/project/_/settings) for your project.
+1. Install the [Google Cloud SDK](https://cloud.google.com/sdk/).
+1. Setup a MongoDB server:
```
Comment (jonparrott) [inline appengine/parse-server/README.md]: Can you say something like "here are two options"? I can imagine someone doing both.

**Comment by jonparrott** [comment]
Lgtm with very minor nit.

**Comment by otymartin** [comment]
@jmdobry How do we update node and parse server?

**Comment by jmdobry** [comment]
In your app's `package.json`you control the version of Node.js and `parse-server` that you use: `"engines"` controls the version of Node.js used and `"dependencies"` controls the version of `parse-server` that you use.

**Comment by otymartin** [comment]
@jmdobry Why do i get this error even if i set `"node": ~4.3` in engines
`parse-server@2.2.1: wanted: {"node":">=4.3"} (current: {"node":"4.2.3","npm":"2.14.7"})`

**Comment by JustinBeckwith** [comment]
Can you share your full `package.json` with us?  

There was a bug in older Cloud SDK versions that may have caused node version mismatch error.  Can you run `gcloud components update`, and then try again?

**Pull request merged**